src/DTO/CustomerDTO.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+namespace App\DTO;
+
+use Symfony\Component\Validator\Constraints as Assert;
+
+class CustomerDTO
+{
+    #[Assert\NotBlank]
+    public string $firstname;
+
+    #[Assert\NotBlank]
+    public string $lastname;
+
+    #[Assert\NotBlank]
+    #[Assert\Email(
+        message: 'The email {{ value }} is not a valid email.',
+    )]
+    public string $email;
```

Comment:
You should add here you custom contraint to validate email already exist
```suggestion
    #[EmailNotExist]
    public string $email;
```